.travis.yml

```diff
@@ -74,11 +74,9 @@ before_install:
 sudo: required
 after_success:
   - codecov
-  - if [ $TRAVIS_BRANCH == "master" ]; then
-      pip install virtualenv;
-      virtualenv ~/env;
-      source ~/env/bin/activate;
-      pip install transifex-client;
-      sudo echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"$TRANSIFEX_USER"$'\npassword = '"$TRANSIFEX_PASSWORD"$'\ntoken = '"$TRANSIFEX_API_TOKEN"$'\n' > ~/.transifexrc;
-      tx push --source --no-interactive;
-      fi
+  - pip install virtualenv
```

Comment:
It should be mentioned in comments that these commands are used for transifex source files push and following link should be given as a reference: 
https://docs.transifex.com/integrations/github#section-integrating-with-travis-ci